### PR TITLE
Added Count up for Why armada page

### LINKED ISSUE #51: Make numbers count up when page is loaded
on the "/exhibitor" page, make the numbers count up fast when the page is loaded. 

Image

### PR DESCRIPTION
**Pull request by AmiyaSX**
https://github.com/armada-ths/armada.nu/assets/71491866/35bfa0d4-d3be-4b11-a04a-9f5ce6a3b36e

### CODE REVIEW

File: pnpm-lock.yaml
Comment (AmiyaSX) [inline pnpm-lock.yaml:1]: I honestly don't know why this file changed a lot, always become nervous when lock file changed...

**Comment by hampfh** [comment]
Looks good! 

I like the effect! A few comments:

CleanShot 2024-05-21 at 10 36 07@2x

For example when the + is added to the 20000 

https://github.com/armada-ths/armada.nu/assets/30767818/f8f5e447-3946-4d54-9c21-76c8f0e55a58

the text jumps to the left, try to prevent layout shifts if possible

**Comment by hampfh** [comment]
I also think the animation can be a little faster, I think now it is 2-3 seconds, see if you can reduce that slightly

File: pnpm-lock.yaml
Comment (AugustPaulsrud) [reply pnpm-lock.yaml:1]: It is good that you bring this up, we'll look into why this is happening. The code still works great, so it should not block it from being merged.

File: src/app/exhibitor/_components/NumberCountUp.tsx
Comment (AugustPaulsrud) [inline src/app/exhibitor/_components/NumberCountUp.tsx:1]: This is  a great reusable component that I think we'll use on more places in the page! In the future we might want to use it for other pages, maybe on the student page to say "100 exhibitors". To make this component more reusable, you could change the "isVisit" and "isDays" into one string argument that will always be appended to the CountUp. Then we could use it for "100 exhibitors" or similar purposes without adding another boolean argument.

**Review by AugustPaulsrud — APPROVED** [review]
Great work, looks nice and the component will definitely be reused! I have commented on one possible change to make in the component code, you can choose to implement this or keep it the current way.

**Pull request merged**